packages/commonwealth/client/scripts/views/pages/NotificationSettings/CommentSubscriptionEntry.tsx

```diff
@@ -96,17 +96,11 @@ export const CommentSubscriptionEntry = (
       </div>
       <div>
         <CWText type="h4" fontWeight="semiBold">
-          <Link to={threadUrl}>
-            <QuillRenderer
-              doc={safeTruncateBody(decodeURI(comment.text))}
-              cutoffLines={4}
-              customShowMoreButton={
-                <CWText type="b1" className="show-more-btn">
-                  Show more
-                </CWText>
-              }
-            />
-          </Link>
+          <QuillRenderer
+            doc={safeTruncateBody(decodeURI(comment.text))}
+            cutoffLines={4}
+            customShowMoreButton={<></>}
```

Comment:
lets remove

Reply:
I can't remove unfortunately. If we remove I get the default show more button and that breaks the workflow.